css-view-transitions-2/Overview.bs

```diff
@@ -299,29 +330,29 @@ The <dfn attribute for=PageRevealEvent>viewTransition</dfn> [=getter steps=] are
 	<div algorithm="monkey patch to rendering">
 		Run the following step in [=update the rendering|updating the renedering=], before [=running the animation frame callbacks=]:
```

Comment:
@bokand to review the timing for this.

Reply:
Let's review this as part of https://github.com/whatwg/html/pull/9791